packages/core-mobile/app/contexts/SendNFTContext.tsx

```diff
@@ -134,13 +135,15 @@ export const SendNFTContextProvider = ({
         })
         .catch(reason => {
           setSendStatus('Fail')
-          showTransactionErrorToast({
-            message: getJsonRpcErrorMessage(reason)
-          })
-          AnalyticsService.capture('NftSendFailed', {
-            errorMessage: reason?.error?.message,
-            chainId: activeNetwork.chainId
-          })
+          if (!isUserRejectedError(reason)) {
+            showTransactionErrorToast({
+              message: getJsonRpcErrorMessage(reason)
+            })
+            AnalyticsService.capture('NftSendFailed', {
+              errorMessage: reason?.error?.message,
+              chainId: activeNetwork.chainId
+            })
+          }
```

Comment:
we were reporting and showing toast even for the user rejected error. this fixes it.